front/src/common/Map/Layers/GeoJSONs.tsx

```diff
@@ -418,7 +418,7 @@ interface EditorSourceProps {
 export const EditorSource = ({ id, data, layers, layerOrder }: EditorSourceProps) => {
   const dataFingerPrint =
     data.type === 'FeatureCollection'
-      ? data.features.map((f) => f.properties?.id).concat()
+      ? data.features.map((f) => f.properties?.id as unknown).concat()
```

Comment:
Here `f.properties` is `any`. We don't actually care about the real type so we can just use `unknown`.